### ISSUE TITLE
Feedback feature: screenshots

### ISSUE THREAD

**Issue by jonkerz** [calacademy-research/antcat#124]
Problem: High barriers-to-entry to send feedback = fewer visitors send feedback/corrections. It may sound silly, but a plain email link really is a "high barrier".

Related problem: People have not been given the opportunity to comment on the new design + other changes.

Solution: A user-friendly feedback form (but also show the old link).

Visitor without an account:
![01_not_logged_in](https://cloud.githubusercontent.com/assets/1513381/14022918/42b287a4-f1e2-11e5-9653-16fb7ff0915a.png)

Logged in user:
![02_logged_in_user](https://cloud.githubusercontent.com/assets/1513381/14022916/42ae8e7e-f1e2-11e5-9931-55524fc88030.png)

Thank-you message:
![03_feedback_sent_notice](https://cloud.githubusercontent.com/assets/1513381/14022919/42b429b0-f1e2-11e5-8ef2-d7f3a221657c.png)

Generated email:
![04_email](https://cloud.githubusercontent.com/assets/1513381/14022917/42b28812-f1e2-11e5-982c-b2f35aaa655e.png)

"New" badge:
![05_new_badge](https://cloud.githubusercontent.com/assets/1513381/14048507/f0b785d0-f2ae-11e5-91a6-64588e34acee.png)
).

Basically:
* Open the form by clicking on Feedback.
* Available text fields: name, email, comment, page. All except comment are optional, so that visitors are more likely to send feedback/corrections.
* The old email link is still linked (visible after opening the form).
* Anyone can send feedback, including users without an account (but they are rate-throttled to 3 feedback messages / every 5 minutes).
* The fields for email and name are prefilled when the user is logged in.
* The page field defaults to the URL from where the feedback modal was opened, but it is also optional.
* The comments are saved in the database and can be browsed by superadmins from the Admin Dashboard.

**Comment by jonkerz** [comment]
Update:
* Emails are forwarded to users flagged in the database (currently only editable from the Admin Dashboard by a superadmin)
* Editors can browse all submission from antcat.org/feedback (linked from the Editor's Panel).

**Comment by jonkerz** [comment]
### TODO
- [ ] Enable comments ([see](https://cloud.githubusercontent.com/assets/1513381/14361440/251f5054-fcfb-11e5-879e-af373d840f6e.png))
- [ ] Make feedback items "closable". Intended workflow: a visitor submits a correction; if it's a complicated issue, add it as an open task and comment on the feedback "I opened a new task for this, please see..." (only visible to editors); once resolved, email the visitor and thank them for their input and mark the feedback as "closed".
